src/Symfony/Bridge/Doctrine/ManagerRegistry.php

```diff
@@ -50,7 +51,7 @@ protected function resetService($name)
         $manager = $this->container->get($name);
 
         if (!$manager instanceof LazyLoadingInterface) {
-            throw new \LogicException('Resetting a non-lazy manager service is not supported. '.(interface_exists(LazyLoadingInterface::class) ? sprintf('Declare the "%s" service as lazy.', $name) : 'Try running "composer require symfony/proxy-manager-bridge".'));
+            throw new \LogicException('Resetting a non-lazy manager service is not supported. '.(interface_exists(LazyLoadingInterface::class) && class_exists(RuntimeInstantiator::class) ? sprintf('Declare the "%s" service as lazy.', $name) : 'Try running "composer require symfony/proxy-manager-bridge".'));
```

Comment:
because LazyLoadingInterface is a required dependency of RuntimeInstantiator, I suppose we can check only the 2nd?

Reply:
That was my first idea (replace rather than augment), but on second thoughts I chose to play it safe and keep the existence check of the actually `instanceof`-tested interface (which also makes it clear that the code handles the possibility of it *not* existing).
But I have no strong opinion... However I won't be able to amend the PR (nor to rebase it to 4.3) soon (sorry I can only work via the web UI currently), but edits from maintainers are allowed ;)